html-tables/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>HTML Tables</title>
+    </head>
+
+    <body>
+        <h1>Student Grade Table</h1>
+
+        <table>     
+            <thead>
+                <tr>
+                <th>Student Name</th>
+                <th>Course</th>
+                <th>Grade</th> 
+                </tr>
+            </thead>
+
+            <tbody>
+                    <tr>
+                        <td>Grace Hopper</td>
+                        <td>Computer Science</td>
+                        <td>100</td>
+                    </tr>
+            </tbody>
+
+            <tbody>
+                <tr>
+                    <td>Alan Turing</td>
+                    <td>Mathematics</td>
+                    <td>100</td>
+                </tr>
+            </tbody>
+
+            <tbody>
+                <tr>
+                    <td>Douglas Crockford</td>
+                    <td>JSON</td>
+                    <td>98</td>
+                </tr>
+            </tbody>
+
+            <tbody>
+                <tr>
+                    <td>Tim Berners-Lee</td>
+                    <td>HTML</td>
+                    <td>9001</td>
+                </tr>
+            </tbody>
+
+            <tbody>
+                <tr>
+                    <td>Linus Torvalds</td>
+                    <td>Git</td>
+                    <td>90</td>
+                </tr>
+            </tbody>
```

Comment:
Tables should have only one `tbody` element.  You can put the rest of the rows in there.